### PR TITLE
fix package paths for maccatalyst and linux

### LINKED ISSUE #346: 🐞 Nuget - maccatalyst missing hacl
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

🐞 Nuget - maccatalyst missing hacl

### Expected Behavior

_No response_

### Steps To Reproduce

_No response_

### Environment

```markdown
- OS:
```

### Anything else?

_No response_

### PR DESCRIPTION
**Pull request by AddressXception**
[//]: # (🚨 Please review the CONTRIBUTING.md in this repository. 💔Thank you!)

### Issue
*Link your PR to an issue*

Fixes #346

### Description
*Please describe your pull request.*

### Testing
*Describe the best way to test or validate your PR.*

### CODE REVIEW

**Review by SteveMaier-IRT — APPROVED** [review]
lgtm

File: bindings/netstandard/ElectionGuard/ElectionGuard.Decryption.Tests/ElectionGuard.Decryption.Tests.csproj [hunk 1]
```diff
@@ -26,6 +25,7 @@
   <ItemGroup>
     <ProjectReference Include="..\ElectionGuard.Decryption\ElectionGuard.Decryption.csproj" />
     <ProjectReference Include="..\ElectionGuard.ElectionSetup\ElectionGuard.ElectionSetup.csproj" />
+    <ProjectReference Include="..\ElectionGuard.Encryption\ElectionGuard.Encryption.csproj" />
```
Comment (AddressXception) [inline bindings/netstandard/ElectionGuard/ElectionGuard.Decryption.Tests/ElectionGuard.Decryption.Tests.csproj:28]: in order to get this to work, i had to revert the use of the nuget package in the UI project and all the way down the stack. it was causing too many issues with resolving assemblies. it definitely makes sense to add it back when we can publish electionsetup and decryption as their own packages for reference by the application

File: bindings/netstandard/ElectionGuard/ElectionGuard.Decryption/ChallengeResponse/BallotChallengeResponse.cs [hunk 2]
```diff
@@ -39,10 +37,10 @@ public record BallotChallengeResponse
     /// </summary>
     public ElementModQ Coefficient { get; init; }
 
-    public Dictionary<string, ContestChallengeResponse> Contests { get; init; } = default!;
-
-    [JsonConstructor]
-    public BallotChallengeResponse(
+    public Dictionary<string, ContestChallengeResponse> Contests { get; init; } = default!;
```
Comment (AddressXception) [inline bindings/netstandard/ElectionGuard/ElectionGuard.Decryption/ChallengeResponse/BallotChallengeResponse.cs:40]: line endings / formatting

File: src/electionguard-ui/ElectionGuard.UI/ElectionGuard.UI.csproj [hunk 3]
```diff
@@ -215,6 +218,38 @@
...
+    </None>
+
+    <None Name="MacCatalyst" Visible="false" Include="$(ElectionGuardLibs)\MacCatalyst\arm64\$(Configuration)\src\*.dylib" >
+			<CopyToOutputDirectory Condition="$([MSBuild]::GetTargetPlatformIdentifier('$(TargetFramework)')) == 'maccatalyst'">PreserveNewest</CopyToOutputDirectory>
+		</None>
+    <None Name="MacCatalyst HACL" Visible="false" Include="$(ElectionGuardLibs)\MacCatalyst\arm64\$(Configuration)\libs\hacl\*.dylib" >
+			<CopyToOutputDirectory Condition="$([MSBuild]::GetTargetPlatformIdentifier('$(TargetFramework)')) == 'maccatalyst'">PreserveNewest</CopyToOutputDirectory>		
+		</None>
+	</ItemGroup>
+	<!-- <Target Name="Copy Windows x64" AfterTargets="Build">
```
Comment (AddressXception) [inline src/electionguard-ui/ElectionGuard.UI/ElectionGuard.UI.csproj:239]: ill remove the commented out code as part of another pr after we cut a release

**Review by john-s-morgan — APPROVED** [review]

**Pull request merged**